src/license.rs

```diff
@@ -0,0 +1,49 @@
+use std::fs;
+use std::io::{BufRead, BufReader};
+use std::path::Path;
+
+use crate::types::CheckResult;
+
+fn check_if_license_is_present(path: &Path) -> CheckResult<bool> {
+    let f =
+        fs::File::open(path).map_err(|e| format!("Failed to open `{}`: {}", path.display(), e))?;
+    let f = BufReader::new(f);
+    let header: Vec<String> = f.lines().take(2).map(|x| x.unwrap()).collect();
+    if header.len() != 2 {
+        println!("xx> Missing header in `{}`", path.display());
+        Ok(false)
+    } else if header[0]
+        != "// Take a look at the license at the top of the repository in the LICENSE file"
```

Comment:
```suggestion
        != "// Take a look at the license at the top of the repository in the LICENSE file."
```

Finally I can also tell you to put a period at the end of your sentences ;)